src/modules/remove_from_list.js

```diff
@@ -0,0 +1,25 @@
+import { Store, AddItemsToList } from './index';
+
+class RemoveItemFromList {
+  static removeItemFromList = (itemList) => {
+    itemList.parentElement.remove();
+    const tasks = Store.getItems();
+    const newTasks = tasks.filter(
+      // eslint-disable-next-line comma-dangle
+      (item) => parseInt(itemList.previousElementSibling.id, 10) !== item.index
+    );
+    Store.setItems(newTasks);
+  };
+
+  static changeTaskIndex = () => {
+    const tasks = Store.getItems();
+    const newTasks = tasks.map((task, index) => {
+      task.index = index;
```

Comment:
- [OPTIONAL] Kindly make sure if the user deleted one task the index of the task should update and starts to index 1.
```suggestion
      task.index = index + 1;
```